src/main/kotlin/solve/settings/visualization/fields/view/VisualizationSettingsLayerCell.kt

```diff
@@ -0,0 +1,240 @@
+package solve.settings.visualization.fields.view
+
+import javafx.application.Platform
+import javafx.geometry.Pos
+import javafx.scene.Node
+import javafx.scene.Parent
+import javafx.scene.Scene
+import javafx.scene.control.Button
+import javafx.scene.control.ComboBoxBase
+import javafx.scene.control.Label
+import javafx.scene.image.Image
+import javafx.scene.paint.Color
+import javafx.scene.text.Font
+import javafx.util.Duration
+import org.controlsfx.control.PopOver
+import solve.scene.controller.SceneController
+import solve.scene.model.LandmarkType
+import solve.scene.model.LayerSettings
+import solve.settings.visualization.popover.PointLayerSettingsPopOverNode
+import solve.utils.*
+import solve.utils.nodes.DragAndDropListCell
+import solve.utils.structures.Point
+import tornadofx.*
+
+class VisualizationSettingsLayerCell(
+    private val sceneController: SceneController
+) : DragAndDropListCell<LayerSettings>() {
```

Comment:
How it can do anything on drag over? Now it does nothing on drag drop. Call `Scene.changeLayerIndex` function through `SceneController` on drag over.
![hRuybD3vKF](https://user-images.githubusercontent.com/55746901/222922491-b5074bd5-7453-4ac6-bead-87d7d2715efc.gif)

Reply:
Still actual